workshop/content/docs/api-deep-dive/features.md

```diff
@@ -174,6 +174,102 @@ the request
 A client application can then retrieve the GeoJSON document and display
 or process it.
 
+### Client Usage
+
+Various clients/softwares supports OGC API- Features, You can checkout latest list [here](https://github.com/opengeospatial/ogcapi-features/blob/master/implementations/clients/README.md)
+
+In this workshop we'll check a client example ( leaflet and Openlayers ), Software example (QGIS), Native API example (GDAL) and also server example ( PygeoAPI )
+
+#### Leaflet Implementation
+
+Leaflet supports reading and implementing GeoJSON by default, thus just like any other  file or API which spits out data as GeoJSON, OGC API - Features can also be expose data as GeoJSON, by passing `f=json` in the string and then it can be utilized in Front end my adding following code block
```

Comment:
- "via the front end by adding the following code:"
- always have colon on text before starting a code block